aspnetcore/mvc/views/tag-helpers/built-in/partial-tag-helper.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Partial Tag Helper
+author: scottaddie
+description: Discover the ASP.NET Core Partial Tag Helper and the role each of its attributes play in rendering a partial view.
+manager: wpickett
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 02/22/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: mvc/views/tag-helpers/builtin-th/partial-tag-helper
+---
+# Partial Tag Helper
+
+By [Scott Addie](https://github.com/scottaddie)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/mvc/views/tag-helpers/built-in/samples) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+The Partial Tag Helper is supported as of ASP.NET Core 2.1. It offers an alternative to [HTML Helper syntax](xref:mvc/views/partial#referencing-a-partial-view) for rendering a [partial view](xref:mvc/views/partial) in Razor Pages or MVC.
```

Comment:
I think this works better as a list:

* require 2.1 or later.
* has an HTML alternative ...